### DISCUSSION TITLE
[2021.09.02] 동물 수 구하기

### DISCUSSION THREAD

**Discussion by hyk339** [pyunji/Hyundai-ITnE-Team_QuadCore#99] [SQL]
[동물 수 구하기](https://programmers.co.kr/learn/courses/30/lessons/59406)

**Comment by hyk339** [comment]
[동물 수 구하기 solved](https://github.com/hyk339/Hyundai-ITnE-Team_QuadCore/blob/master/khy/sql/PR_59406.sql)

**Comment by pyunji** [comment]
[동물 수 구하기 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/pyunji/pyunji/sql/PR_59406.sql)

**Comment by chaeyeon4u** [comment]
[동물 수 구하기 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/ycy/homework/SQL/PR_59406.sql)

**Comment by leechang4** [comment]
[동물 수 구하기 solved](https://github.com/leechang4/Hyundai-ITnE-Team_QuadCore/blob/master/lch/SQL/PR_59406.sql)
